addOns/ascanrulesBeta/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 ### Changed
+- XML External Entity Attack scan rule changed to detect a possible XML File Reflection Attack when XML validation is present and limited the 
+  number of requests sent depending on the strength at which the rule is invoked.(Issue 6204)
```

Comment:
I'd remove `and limited the number of requests sent depending on the strength at which the rule is invoked.` and move it to the help (i.e. mention that strength is used when testing each tag individually).